in-progress/gallery.yml

```diff
@@ -0,0 +1,141 @@
+# Installation notes:
+# - Set up OME RHEL 7 machine.
+# - Set up web server TLS in resources/certificates:
+#   star_openmicroscopy_org.crt+bundle, star_openmicroscopy_org.key
+# - Set host variables:
+#   database_password, server_password
+# - Run playbook.
+# - systemctl stop omero-{server,web}
+# - Migrate binary repository to /OMERO/ with rsync.
+# - Migrate database with pg_dump, pg_restore.
+# - Upgrade database.
+# - bin/omero db password for root user.
+# - systemctl start omero-{server,web}
+
+- hosts: omero-sls-gallery
+  roles:
+    - role: ome.network
+      network_ifaces:
+        - device: ens192
+          type: "Ethernet"
+          ipv6init: "no"
+          prefix: 21
+          ip: 10.0.50.12
+          gateway: 10.0.55.254
+          dns1: 10.1.255.216
+          dns2: 10.1.255.226
+          network_additional_params:
+            name: "ens192"
+            ipv4_failure_fatal: "no"
+            ipv6_autoconf: "yes"
+            ipv6_defroute: "yes"
+            ipv6_peerdns: "yes"
+            ipv6_peerroutes: "yes"
+            ipv6_failure_fatal: "no"
+            ipv6_addr_gen_mode: "stable-privacy"
+            defroute: "yes"
+            domain: openmicroscopy.org
+
+    - role: ome.lvm-partition
+      lvm_vgname: VolGroup00
+      lvm_lvname: 'var_lib_psql'
+      lvm_lvmount: '/var/lib/pgsql'
+      lvm_lvsize: 8G
+      lvm_lvfilesystem: xfs
+      lvm_shrink: False
+
+    - role: ome.postgresql
+      postgresql_databases:
+        - name: omero
+      postgresql_users:
+        - user: omero
+          password: "{{ database_password }}"
+          databases: [omero]
+      postgresql_version: "10"
+
+    - role: ome.lvm-partition
+      lvm_vgname: VolGroup00
+      lvm_lvname: 'omero'
+      lvm_lvmount: '/OMERO'
+      lvm_lvsize: 80G
+      lvm_lvfilesystem: xfs
+      lvm_shrink: False
+
+    - role: ome.omero-server
+      ice_python_wheel: https://github.com/ome/zeroc-ice-py-centos7/releases/download/0.1.0/zeroc_ice-3.6.4-cp27-cp27mu-linux_x86_64.whl
+      omero_server_systemd_start: False
+      omero_server_release: latest
+      omero_server_dbpassword: "{{ database_password }}"
+      omero_server_rootpassword: "{{ server_password }}"
+      omero_server_config_set:
+        omero.client.ui.menu.dropdown.colleagues.enabled: False
+        omero.client.ui.menu.dropdown.everyone.label: "All images"
+        omero.client.ui.menu.dropdown.leaders.label: "Gallery:"
+        omero.db.poolsize: 50
+        omero.jvmcfg.percent.blitz: 50
+        omero.jvmcfg.percent.indexer: 20
+        omero.jvmcfg.percent.pixeldata: 30
+        omero.ldap.config: True
+        omero.ldap.base: ou=lifesci,o=dundee
```

Comment:
Can you put this internal LDAP configuration into a private host/group_vars file instead of the public playbook?

Reply:
Done on https://github.com/openmicroscopy/management_tools/pull/1047.